app/assets/stylesheets/tweets.css.scss

```diff
@@ -0,0 +1,10 @@
+.tweet {
+  height: 200px;
+  position: relative;
+}
+
+.tweet-footer {
+  position: absolute;
+  bottom: 0;
+  width: 87%;
```

Comment:
why 87%?  why not 85%?
can it be changed? I mean CSS.

you can try to use "flex-box" CSS styling.

OR http://prntscr.com/i8dmij this card